auth/src/test/kotlin/gov/cdc/prime/reportstream/auth/controller/HealthControllerTest.kt

```diff
@@ -0,0 +1,32 @@
+package gov.cdc.prime.reportstream.auth.controller
+
+import gov.cdc.prime.reportstream.auth.AuthApplicationConstants
+import gov.cdc.prime.reportstream.auth.model.ApplicationStatus
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.reactive.AutoConfigureWebTestClient
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.security.test.web.reactive.server.SecurityMockServerConfigurers.csrf
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import org.springframework.test.web.reactive.server.WebTestClient
+import kotlin.test.Test
+
+@ExtendWith(SpringExtension::class)
+@SpringBootTest
+@AutoConfigureWebTestClient
+class HealthControllerTest @Autowired constructor(
+    private val webTestClient: WebTestClient
+) {
+
+    @Test
+    fun `successful healthcheck`() {
```

Comment:
Added a simple test here just to make sure project is compiling and healthcheck endpoint is functional.